package.json

```diff
@@ -2,7 +2,7 @@
   "name": "suomifi-design-system",
   "private": true,
   "description": "Suomi.fi Design System",
-  "version": "0.2.0",
+  "version": "1.0.0",
   "author": "Jari Porola <jari.porola@gmail.com>",
```

Comment:
Would this be good place to update also author/contributors for the repo?

Reply:
Not at this point. During the next change maybe as this PR is only merge from dev to master.